### PR TITLE
[project-test] Add readable and writable to temple test

### PR DESCRIPTION
**Pull request by jaylees14**
Adds readable and writable tests to `temple test`. Everything seemed to work, so very happy 😄

### CODE REVIEW

File: src/main/scala/temple/test/internal/CRUDServiceTest.scala [hunk 1]
```diff
@@ -16,6 +20,56 @@ class CRUDServiceTest(
   usesAuth: Boolean,
 ) extends ServiceTest(serviceName, baseURL, usesAuth) {
 
+  private def serviceReadable: Metadata.Readable =
+    service.lookupLocalMetadata[Metadata.Readable].getOrElse {
+      if (usesAuth) Metadata.Readable.This else Metadata.Readable.All
```
Comment (c-harding) [inline src/main/scala/temple/test/internal/CRUDServiceTest.scala]: doesn’t ProjectConfig have something for this?

File: src/main/scala/temple/test/internal/CRUDServiceTest.scala [hunk 2]
```diff
@@ -16,6 +20,56 @@ class CRUDServiceTest(
...
+    }
+
+  private def serviceWritable: Metadata.Writable =
+    service.lookupLocalMetadata[Metadata.Writable].getOrElse {
+      if (usesAuth) Metadata.Writable.This else Metadata.Writable.All
```
Comment (c-harding) [inline src/main/scala/temple/test/internal/CRUDServiceTest.scala]: ditto

**Review by c-harding — APPROVED** [review]
very nice

File: src/main/scala/temple/test/internal/CRUDServiceTest.scala (hunk 1 above)
Comment (jaylees14) [reply src/main/scala/temple/test/internal/CRUDServiceTest.scala]: Ah so it does! Thanks

**Review by Lewiky — APPROVED** [review]
LGTM

**Pull request merged**